contracts/TopShotMarketV3.cdc

```diff
@@ -237,18 +237,6 @@ pub contract TopShotMarketV3 {
             panic("No token matching this ID for sale!")
         }
 
-        /// changePercentage changes the cut percentage of the tokens that are for sale
-        ///
-        /// Parameters: newPercent: The new cut percentage for the sale
-        pub fun changePercentage(_ newPercent: UFix64) {
```

Comment:
I am of the opinion that this shouldn't be removed, but we'll wait for @EricLin2004 's response